OpenRA.Mods.Common/Traits/DeliversExperience.cs

```diff
@@ -0,0 +1,127 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Drawing;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Mods.Common.Orders;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Give experience actor has to other actors with the `AcceptsDeliveredExperience` trait.")]
```

Comment:
This description is unclean. "This actor can grant experience levels equal to it's own current level via entering to other actors with the AcceptsDeliveredExperience trait." perhaps?